.travis.yml

```diff
@@ -155,8 +155,5 @@ script:
 
 after_success:
     - if [[ $SETUP_CMD == *--coverage* ]]; then
-        if [ "$TRAVIS_REPO_SLUG" = "astropy/astropy" -o "$TRAVIS_PULL_REQUEST_SLUG" = "astropy/astropy" ]; then
-          cpp-coveralls -E ".*convolution.*" -E ".*_erfa.*" -E ".*\.l" -E ".*\.y" -E ".*flexed.*" -E ".*cextern.*" -E ".*_np_utils.*" -E ".*cparser.*" -E ".*cython_impl.*" --dump c-coveralls.json;
-          coveralls --merge=c-coveralls.json --rcfile='astropy/tests/coveragerc';
-        fi;
+        codecov --gcov-glob "*cextern*";
```

Comment:
This would appear to be changing what's getting ignored vs. coveralls (which ignored convolution, _erfa,and some of the other stuff as the diff shows)? Or is it that codecov does a better job of recognizing some of the C-stuff vs coveralls?

Reply:
my understanding is that codecov is much better with C